Pop Alexia/matrice y/main.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    int matrix[3][3],i,j;
+    for (i=0; i<3; i++)
+    {
+        for(j=0; j<3; j++)
+        {
+            scanf("%d",&matrix[i][j]);
+        }
+
+    }
+    printf("The secondary diagonal is : \n");
+    for(i=0; i<3; i++)
+    {
+        printf("%d",matrix[i][2-i]);
```

Comment:
well, yes, in case you have a `[3][3]` matrix, this would be the second diagonal, but if you had a different size?
You cannot expect to modify the `[2-i]` each time you change the size, find something more generic